packages/gcp/src/lib/Recommendations.ts

```diff
@@ -253,7 +253,7 @@ export default class Recommendations implements ICloudRecommendationsService {
             timestamp: undefined,
           }
           resourceDetails = {
-            resourceId: instanceDetails.id,
+            resourceId: instanceDetails.id.toString(),
```

Comment:
Assuming this needed to be updated from new typing issues?

Reply:
Yup! The new interface references it as a `string | number`